src/redux/actions/cellSets/createCellSet.js

```diff
@@ -18,6 +18,7 @@ const createCellSet = (experimentId, name, color, cellIds) => async (dispatch, g
     name,
     color,
     cellIds: new Set([...cellIds].map((id) => parseInt(id, 10))),
+    type: 'cellSets',
```

Comment:
This field can be used to differntiate `cellSets` properties from `metadataCategorical` properties. All cell sets created from QC have this property. Newly created `cellSets` did not have this, so I added it to facilitate easier filtering of `cellSets` to be used in the embedding.